### PR TITLE
[Wizard] General: Gradle Tests dont work #224

### PR DESCRIPTION
**Pull request by holgerschill**
Signed-off-by: Holger Schill <Holger.Schill@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext/build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ dependencies {
 	compile project(':org.eclipse.xtext.util')
 	compile "org.eclipse.xtend:org.eclipse.xtend.lib:$versions.xtext"
 	compile "log4j:log4j:$versions.log4j"
-	compile "org.eclipse.equinox:org.eclipse.equinox.common:$versions.equinoxCommon"
+	compile "org.eclipse.platform:org.eclipse.equinox.common:$versions.equinoxCommon"
 	compile "org.eclipse.emf:org.eclipse.emf.common:$versions.emfCommon"
 	compile "org.eclipse.emf:org.eclipse.emf.ecore.xmi:$versions.emfEcore"
 	compile "com.google.inject:guice:$versions.guice"
```
Comment (spoenemann) [inline org.eclipse.xtext/build.gradle]: To be consistent, let's also update equinox.registry (line 14) to the new group id: `org.eclipse.platform:org.eclipse.equinox.registry:3.6.100`

Comment (holgerschill) [inline org.eclipse.xtext/build.gradle]: Done

**Comment by holgerschill** [comment]
http://services.typefox.io/open-source/jenkins/job/xtext-core/job/hsIssue224/3 looks good now.

**Comment by spoenemann** [comment]
Looks good to me.

**Pull request merged**